### PR TITLE
marketing link

### PR DESCRIPTION
**Pull request by Thobeats**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: app/Http/Controllers/AssistantController.php [hunk 1]
```diff
@@ -357,7 +314,7 @@ public function update(Request $request, $id)
             }
 
         } catch ( \Exception $e ) {
-            $data = json_decode($e->getBody()->getContents());
+            dd($e);
```
Comment (eni4sure) [inline app/Http/Controllers/AssistantController.php:317]: Please remove this dd

File: app/Http/Controllers/AssistantController.php [hunk 2]
```diff
@@ -259,7 +259,7 @@ public function edit($id)
             $data = json_decode($response->getBody());
 
             if ( $response->getStatusCode() == 200 ) {
-                return view('backend.assistant.update')->with('response', $data->data->assistants);
+                return view('backend.assistant.update')->with('response', $data->data->assistantData);
```
Comment (eni4sure) [inline app/Http/Controllers/AssistantController.php:262]: Hope this corresponds with the API ??

File: resources/views/backend/assistant/update.blade.php [hunk 3]
```diff
@@ -55,7 +55,7 @@
                                           </div>
                                         <div class="form-group col-md-6" >
                                             <label for="inputEmailAddress"> Password </label>
```
Comment (eni4sure) [inline resources/views/backend/assistant/update.blade.php:57]: Please update the "for" param

File: app/Http/Controllers/AssistantController.php (hunk 2 above)
Comment (kofimokome) [reply app/Http/Controllers/AssistantController.php:262]: I am working on this assistan controller
 anyway. But the changes will not be live till Friday

File: resources/views/backend/assistant/update.blade.php (hunk 3 above)
Comment (kofimokome) [reply resources/views/backend/assistant/update.blade.php:57]: this view will be deleted when I merge my codes before Friday. 
I've done some refactoring on assistants

File: app/Http/Controllers/AssistantController.php (hunk 1 above)
Comment (kofimokome) [reply app/Http/Controllers/AssistantController.php:317]: Hmmm. Why are doing my work for me?
thank you

Comment (Thobeats) [reply app/Http/Controllers/AssistantController.php:317]: Sorry

File: app/Http/Controllers/AssistantController.php (hunk 2 above)
Comment (Thobeats) [reply app/Http/Controllers/AssistantController.php:262]: YEs it does, that is why i used dd()

**Pull request merged**